4.0/entrypoint.sh

```diff
@@ -47,4 +47,6 @@ psql $POSTGRES_DB -h $POSTGRES_HOST -p $POSTGRES_PORT  -U $POSTGRES_USER -c "UPD
 psql $POSTGRES_DB -h $POSTGRES_HOST -p $POSTGRES_PORT  -U $POSTGRES_USER -c "UPDATE vdc_options set option_value = '$HOST_INSTALL' where option_name = 'InstallVds';"
 psql $POSTGRES_DB -h $POSTGRES_HOST -p $POSTGRES_PORT  -U $POSTGRES_USER -c "UPDATE vdc_options set option_value = '$HOST_USE_IDENTIFIER' WHERE option_name = 'UseHostNameIdentifier';"
 
+engine-config -s BlockMigrationOnSwapUsagePercentage=99999
```

Comment:
@rmohr  wrote in the commit msg - ""To allow running vm on hosts we don't want to fail the vm if swap memory
usage is high.""
This is a policy check that prevent the vm from running in the node container
the equation is:  swap_total - swap_free - mem_available * 100 / physical_mem_mb <= BlockMigrationOnSwapUsagePercentage

and the default value is BlockMigrationOnSwapUsagePercentage=0.